src/app/app-routing.tsx

```diff
@@ -0,0 +1,45 @@
+import { lazy } from 'react';
+import { Route, Routes } from 'react-router-dom';
+
+import { PrivateRoute } from '@/features/private-route';
+import { TypeAppRoute } from '@/shared/config';
+import { AppLayout } from './app-layout';
+
+const EditorPage = lazy(async () => ({
+  default: (await import('@/pages/editor')).EditorPage,
+}));
+
+const RegisterPage = lazy(async () => ({
+  default: (await import('@/pages/register')).RegisterPage,
+}));
+
+const LoginPage = lazy(async () => ({
+  default: (await import('@/pages/login')).LoginPage,
+}));
+
+const WelcomePage = lazy(async () => ({
+  default: (await import('@/pages/welcome')).WelcomePage,
+}));
```

Comment:
Are you sure you want to lazy load the first page? From my point that's an overkill